roles/container-engine/cri-o/tasks/crictl.yml

```diff
@@ -21,7 +21,14 @@
     group: no
   delegate_to: "{{ inventory_hostname }}"
 
-- name: Install crictl completion
-  shell: "{{ bin_dir }}/crictl completion >/etc/bash_completion.d/crictl"
-  ignore_errors: True
+- name: Get crictl completion
+  shell: "{{ bin_dir }}/crictl completion"
   when: ansible_distribution in ["CentOS","RedHat", "Ubuntu", "Debian"]
+  changed_when: False
+  register: cri_completion
+
+- name: Install crictl completion
+  copy:
+    dest: /etc/bash_completion.d/crictl
+    content: "{{ cri_completion.stdout }}"
+  when: ansible_distribution in ["CentOS","RedHat", "Ubuntu", "Debian"]
```

Comment:
should that not be `when: cri_completion is defined` (aka previous task was executed)

Reply:
Changed in latest commit.